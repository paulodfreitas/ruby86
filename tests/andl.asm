.module entry_code
.pseg
.org 0
        irmovl $c, %eax
        irmovl $6, %edx
        andl %eax, %edx
        halt
.end
