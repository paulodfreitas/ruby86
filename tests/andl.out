@ 0000
30
80
0c
00
00
00
30
82
06
00
00
00
62
02
10
@ 000f
